models/stdp_connection.h

```diff
@@ -185,7 +187,7 @@ class STDPConnection : public Connection< targetidentifierT >
   {
     double_t norm_w =
       ( w / Wmax_ ) - ( alpha_ * lambda_ * std::pow( w / Wmax_, mu_minus_ ) * kminus );
-    return norm_w > 0.0 ? norm_w * Wmax_ : 0.0;
+    return norm_w > 0.0 ? norm_w * Wmax_ : copysign( 0.0, Wmax_ );
```

Comment:
With this line, I wanted to make sure that the sign of `weight` does not flip from negative to positive if it becomes 0. After having another look into this and after testing, I agree that it was unnecessary.
I fixed that in 0ebe4ba3d9d112bf1af7d999f3bd3c0f1789b0ad.